src/http.ts

```diff
@@ -30,6 +28,12 @@ export function customOrigin(
   }
 }
 
+declare global {
+  namespace Express {
+    interface User extends AuthUser {}
+  }
+}
```

Comment:
I had a separate file earlier but also did not know what a good location was for this. Should I just make a `custom-types.d.ts` or `express.d.ts` in the root folder?

Reply:
I think a `types.d.ts` is fine, because in the future we can add extra definitions in the same file as well. As I assume these would be scarce, I see no use in creating separate files (except for creating more clutter).

Reply:
Moved to `types.d.ts`.